Marlin/src/core/boards.h

```diff
@@ -323,9 +323,10 @@
 #define BOARD_CHITU3D_V6              4034  // Chitu3D TronXY X5SA V5 Board
 #define BOARD_CREALITY_V4             4035  // Creality v4.x (STM32F103RE)
 #define BOARD_CREALITY_V427           4036  // Creality v4.2.7 (STM32F103RE)
-#define BOARD_TRIGORILLA_PRO          4037  // Trigorilla Pro (STM32F103ZET6)
-#define BOARD_FLY_MINI                4038  // FLY MINI (STM32F103RCT6)
-#define BOARD_FLSUN_HISPEED           4039  // FLSUN HiSpeedV1 (STM32F103VET6)
+#define BOARD_CREALITY_CR_6_SE        4037  // Creality v4.?.? (STM32F103??)
```

Comment:
v4.5.2 - dated 2020-05-23

Reply:
Thanks! I suppose I could just open up the machine and look….

Reply:
@thinkyhead We've been attempting to [merge the Creality changes](https://github.com/CR6Community/Marlin/commit/f3876e81a74b8ffebdef7776e721f7c25e0ce770) to [bugfix-2.0](https://github.com/CR6Community/Marlin) for a while. 

I'm [attempting to re-implement the touch screen interfacing](https://github.com/CR6Community/Marlin/tree/extui) code on top of extui, but damn it is a lot of work, the way the touch screen is plastered together! This might also be of interest to your: https://github.com/CR6Community/CR-6-touchscreen#README